transit_odp/notifications/templates/notifications/data_end_point_unreachable.txt

```diff
@@ -10,7 +10,7 @@ Short Description: {{ feed_short_description }}
 Action required:
     1) Double check that this is the correct URL for the data you’d like to publish: {{ data_set_url }}
     2) Follow this link to review and if necessary, update the dataset with the correct URL for your bus data: {{ link }}
-    3) Review the notifications received from the Bus Open Data Service to ensure your data is successfully re-published
+    3) Review the notifications received from the Bus Open Data Service to ensure your data is successfully re-published.
```

Comment:
On line 6 & 7, the expected format has "data set / feed" and "data set / feed ID" resp.